pubspec.yaml

```
@@ -6,7 +6,7 @@ documentation: https://docs.datadome.co/docs/flutter-plugin
 
 environment:
   sdk: ^3.0.0
-  flutter: ^3.0.0
+  flutter: ">=3.0.0"
```

won't that be risky when they release flytter v4? 
```suggestion
  flutter: ">=3.0.0 <4.0.0"
```

---

Nope, Flutter stays compatible across majors. You can have more than one version of Flutter in your build.